### ISSUE TITLE
Carbonite.Warehouse\NxWarehouse.lua:771: hooksecurefunc(): SetTradeSkillItem is not a function

### ISSUE THREAD

**Issue by Thrumbar** [Rythal/Carbonite#407]
1x Carbonite.Warehouse\NxWarehouse.lua:771: hooksecurefunc(): SetTradeSkillItem is not a function
[C]: in function `hooksecurefunc'
Carbonite.Warehouse\NxWarehouse.lua:771: in function `func'
Carbonite\Libs\AceTimer-3.0\AceTimer-3.0-17.lua:55: in function <Carbonite\Libs\AceTimer-3.0\AceTimer-3.0.lua:48>

Locals:

**Comment by IrcDirk** [comment]
Are u testing DEV version? Since it has udpated source for Beta Legion it should work for PTR... I dont have Beta access yet and no space left for PTR, but tomorrow Pre-Patch comes live, so i will be making fixes to DEV branch in free time.

**Comment by JWMcQuire** [comment]
I myself was on PTR using the Dev/beta version when I received this error and one other I posted in a new thread I created on WowInterface here: http://www.wowinterface.com/forums/showthread.php?t=53893

I received other errors while fighting Legion forces in Crossroads on my Alliance mage.

However, I only posted this and the other error just to start a thread and to show what I did to fix it temporarily to get the quest log to show up. Also, when I made the temporary edit, the other error I posted went away. Beyond that and some of the other errors I got while in Crossroads Carbonite actually is working quite well in PTR.

I will be more interested in seeing how it looks when Bliz is done pushing the pre-expansion to live WoW today
